allennlp/tests/predictors/decomposable_attention_test.py

```diff
@@ -79,3 +79,39 @@ def test_batch_prediction(self):
             sumexps = sum(exps)
             for e, p in zip(exps, label_probs):
                 assert e / sumexps == approx(p)
+
+    def test_predictions_to_labeled_instances(self):
+        inputs = {
+                "premise": "I always write unit tests for my code.",
+                "hypothesis": "One time I didn't write any unit tests for my code."
+        }
+
+        archive = load_archive(self.FIXTURES_ROOT / 'decomposable_attention' / 'serialization' / 'model.tar.gz')
+        predictor = Predictor.from_archive(archive, 'textual-entailment')
+
+        instance = predictor._json_to_instance(inputs)
+        outputs = predictor._model.forward_on_instance(instance)
+        new_instances = predictor.predictions_to_labeled_instances(instance, outputs)
```

Comment:
Similar checks for the number of new instances here; and it looks like you're missing a test for the sequence tagger.

Reply:
it doesn't look like there is a test for the sequence tagger predictor, should I add one?

Reply:
Yes.  Don't feel like you have to add tests for stuff that you aren't adding in this PR; that's someone else's job.  I mean, you can if you want, but it's not necessary right now.

Reply:
seems like we don't have a serialized model for this yet, how do I add one?

Reply:
Add an appropriate entry here: https://github.com/allenai/allennlp/blob/64d16acffe99fe2beebd0e3629abec297005b24d/scripts/train_fixtures.py#L68 and run that script.  You'll want to comment out all of the models except the one that you want to train, so you don't modify all of the other serialized models.